great_expectations/expectations/metrics/column_pair_map_metrics/column_pair_values_equal.py

```diff
@@ -11,11 +15,19 @@ class ColumnPairValuesEqual(ColumnPairMapMetricProvider):
     domain_keys = ("batch_id", "table", "column_A", "column_B")
     default_kwarg_values = {"ignore_row_if": "both_values_are_missing"}
 
-    # TODO: <Alex>ALEX -- temporarily only a Pandas implementation is provided (others to follow).</Alex>
+    # TODO: <Alex>ALEX -- temporarily only Pandas and SQL Alchemy implementations are provided (Spark to follow).</Alex>
     @column_pair_condition_partial(engine=PandasExecutionEngine)
     def _pandas(cls, column_A, column_B, **kwargs):
         ignore_row_if = kwargs.get("ignore_row_if")
```

Comment:
@NathanFarmer Do we really need the `ignore_row_if` processing here?  It should come from the expectations layer.  Let us delete it from here and similar Metrics-level objects.  Thanks!

Reply:
@alexsherstinsky I agree. I think this is a relic of the old way of handling that argument.